open-banking-brasil-financial-api-1_ID1-ptbr.md

```diff
@@ -38,11 +38,11 @@
       uri = "https://openbankingbrasil.org.br/"
 %%%
 
-.# Prefácio  {#Foreword}
+# Prefácio  {#Foreword}
```

Comment:
This is done deliberately as it is how the XML to RFC tools work (or any other RFC) so please revert.

Reply:
I thought it was a typo... sorry :-) . Reverted on the next commit.